### PR TITLE
[RFR] Delete pm2 apps with makefile

### PR DESCRIPTION
**Pull request by Kmaschta**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -37,6 +37,8 @@ run-dev:
 	@node_modules/.bin/pm2 start ./pm2_servers/dev.json
 stop-dev:
 	@node_modules/.bin/pm2 stop ./pm2_servers/dev.json
+delete-dev:
```
Comment (alexisjanvier) [inline makefile]: You could use `make servers-clear-all` fir that ?

Comment (djhi) [inline makefile]: yup, seems redundant

File: makefile [hunk 2]
```diff
@@ -37,6 +37,8 @@ run-dev:
 	@node_modules/.bin/pm2 start ./pm2_servers/dev.json
 stop-dev:
 	@node_modules/.bin/pm2 stop ./pm2_servers/dev.json
+servers-clear-all:
```
Comment (alexisjanvier) [inline makefile]: My fault, i wasn't clear. It's already done on line 101

Comment (alexisjanvier) [inline makefile]: https://github.com/marmelab/javascript-boilerplate/pull/18

Comment (Kmaschta) [inline makefile]: Aaah !
Erf, yes I do, I'll close this useless PR

**Pull request closed without merging**